### PR TITLE
add toml libray in dependencies

### PR DESCRIPTION
**Pull request by jgirardet**

### CODE REVIEW

File: repository/dependencies.json [hunk 1]
```diff
@@ -1073,6 +1073,20 @@
 				}
 			]
 		},
+		{
+			"name": "toml library",
```
Comment (FichteFoll) [inline repository/dependencies.json]: This should have been `"name": "toml"`, at least according to your readme.

Comment (jgirardet) [reply repository/dependencies.json]: there is already a package named toml so I had to find a new name.
I will change this on my repo.
That said adding `toml library` in dependencies today fails. I do not know if it's due to the space in the name or if it only take a bit time after merge.

Comment (FichteFoll) [reply repository/dependencies.json]: I don't think spaces have usage implications on the curretn setup, although I would strongly recommend avoiding them.

Do you have a link for the `toml` dependency and where it's defined? It's not in `dependencies.json` on this repo, which means it's hard to find as dependencies aren't displayed on the website.

Comment (jgirardet) [reply repository/dependencies.json]: https://packagecontrol.io/packages/TOML
You mean windows is case Insensitive ?
At the end whats you proposal for the name : sublime_toml, toml_library, toml_lib ? should I repush a PR with a new name ?

Comment (FichteFoll) [reply repository/dependencies.json]: I suppose `python-toml`, like we already have `python-markdown` and `python-jinja2`, is the best solution.

Comment (jgirardet) [reply repository/dependencies.json]: done in  #7343

**Pull request merged**